apps/studio/components/interfaces/Realtime/Inspector/MessagesTable.tsx

```diff
@@ -50,7 +51,7 @@ const NoResultAlert = ({
               </Button>
             </div>
             <div className="w-full px-5 py-4 items-center gap-4 inline-flex border-b">
-              <IconPresence size="xlarge" className="text-background bg-foreground rounded w-6" />
+              <Presence size="xlarge" className="text-background bg-foreground rounded w-6" />
```

Comment:
hmm that doesn't look right 😅 
<img width="420" alt="Screenshot 2024-10-14 at 10 28 01" src="https://github.com/user-attachments/assets/926a0285-2115-4931-9aa8-ebab6f39893b">